index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <title>Personal Portfolio</title>
+  <meta name="viewport" content="width=device-width,initial-scale=1" />
+  <link rel="stylesheet" type="text/css" href="assets/css/style.css" />
+</head>
+
+<body>
+
+  <header>
+    <div class="wrapper menu">
+      <span class="menu-logo">My Logo</span>
+      <span class="menu-button"></span>
+    </div>
+  </header>
+
+  <main>
+
+    <section class="wrapper hero">
+      <h1 class="hero-title">Hey There. I’m Alex.</h1>
+      <div class="hero-subtitle">I'm a Software Developer</div>
```

Comment:
:+1: Great work on the font styles of the title! However, it's important to ensure that the appropriate semantic HTML tags are used for different sections of content. In this case, since this `div` is acting as a subtitle, it would be best to use a heading tag to provide proper semantic structure to your content. :grinning:

- [ ] Please make sure to use the appropriate heading tag, such as `<h2>` or `<h3>`, for the title. This will improve the accessibility and understanding of the content structure for both users and search engines. To learn more check this [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#accessibility_concerns) :pray: